docs/index.rst

```diff
@@ -50,8 +50,7 @@ Contents:
 
    prod-requirements
    prod-install
-   prod-auth-first-login
-   prod-health-check-debug
+   prod-troubleshooting
    prod-customize
    prod-maintain-secure-upgrade
    prod-authentication-methods
```

Comment:
Yeah, I'm not sure it's important enough to belong in the top-level ToC at the top level.  I'd add it if there was an easy way to put it at the second level (visible only when you click on "Installation"), though.  Not sure how to do that with rST.